### PR TITLE
DR | Downtime updates & testing

### PR DESCRIPTION
**Pull request by Mottie**
## Summary

- _(Summarize the changes that have been made to the platform)_
  > - Added appropriate header to save-in-progress expired alert
  > - Add appropriate header to downtime message - h2 on intro page and h3 on review & submit
  > - Update downtime active alert to include appropriate header level and app type
  > - Updated unit tests
  > - Added downtime e2e testing to HLR, NOD & Supplemental Claims 
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- _(Which team do you work for, does your team own the maintenance of this component?)_
  > Benefits Decision Reviews
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)

[#79233](https://github.com/department-of-veterans-affairs/va.gov-team/issues/79233)

## Testing done

- _Describe what the old behavior was prior to the change_
  > Incorrect header levels on review & submit downtime alert
  > Downtime alert was hard-coded to show "tool" as the app type
  > No e2e testing of DR downtime notifications
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| No `appType`  |  Screenshot 2024-05-10 at 2 23 17 PM | Screenshot 2024-05-10 at 2 22 18 PM |
| Intro page | Screenshot 2024-05-10 at 9 24 04 AM | Screenshot 2024-05-13 at 2 59 28 PM |
| Review & submit | Screenshot 2024-05-10 at 1 50 48 PM | Screenshot 2024-05-13 at 2 52 07 PM |

## What areas of the site does it impact?

All forms using `SaveInProgressIntro` and downtime services

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/facility-locator/containers/FacilityLocatorApp.jsx [hunk 1]
```diff
@@ -73,6 +73,7 @@ class FacilityLocatorApp extends React.Component {
         </div>
         <DowntimeNotification
           appTitle="facility locator tool"
+          customText={{ appType: 'tool' }}
```
Comment (Mottie) [inline src/applications/facility-locator/containers/FacilityLocatorApp.jsx:76]: This used to be hard-coded as "tool" in the [`Down` component](https://github.com/department-of-veterans-affairs/vets-website/pull/29755/files#diff-2677f12bb1836637d5a6f5d83ce66fc8b3682dff0a18b2a8de7968a330a952e8L12). `customText` is from the `formConfig` setting used by forms.

File: src/applications/check-in/components/layout/Wrapper.jsx [hunk 2]
```diff
@@ -77,6 +77,7 @@ const Wrapper = props => {
         <DowntimeNotification
           appTitle={appTitle}
           dependencies={[downtimeDependency]}
+          customText={{ appType: 'tool' }}
```
Comment (Mottie) [inline src/applications/check-in/components/layout/Wrapper.jsx:80]: This used to be hard-coded as "tool" in the [`Down` component](https://github.com/department-of-veterans-affairs/vets-website/pull/29755/files#diff-2677f12bb1836637d5a6f5d83ce66fc8b3682dff0a18b2a8de7968a330a952e8L12). `customText` is from the `formConfig` setting used by forms.

**Review by brianseek — APPROVED** [review]
👍

**Review by anniebtran — APPROVED** [review]
LGTM

**Review by maxx1128 — APPROVED** [review]

**Review by harshilmoradia — APPROVED** [review]

**Review by eselkin — APPROVED** [review]
LGTM

**Pull request merged**